### ISSUE TITLE
Between values constraint

### ISSUE THREAD

**Issue by pvk-developer** [sdv-dev/SDV#367]
### Problem Description

Currently there is only a GreaterThan constraint, but we can't specify Between  specific values, such as scalar values (i.e: a range of 0 to 10). The current GreaterThan works only with columns

### Expected behavior

If we use the employees demo dataset as  example, let's say that our company has been up for 5 years only, so our employees can only be from 0 to 5 years max:

```python
from sdv.demo import load_tabular_demo

employees = load_tabular_demo()
from sdv.constraints import Between

years_between_constraint = Between(
    low=0,
    high=5,
    column='years_in_the_company'
)

constraints = years_between_constraint

from sdv.tabular import GaussianCopula
gc = GaussianCopula(constraints=constraints)

gc.fit(employees)
...
```

**Comment by npatki** [comment]
I referenced another issue where we're discussing base cases we'll need to build this functionality (and others).

We'll probably want to consider both the open interval (`> and <`) and closed interval (`>= and =<`) as separate cases. Not to mention combinations of that. Ideally, we can prioritize the functionality based on usage.

**Comment by csala** [comment]
Closed in #493
